ts/features/itwallet/common/store/selectors/__tests__/preferences.test.ts

```diff
@@ -70,3 +71,18 @@ describe("itwRequestedCredentialsSelector", () => {
     MockDate.reset();
   });
 });
+
+describe("itwAuthLevelSelector", () => {
+  it("should return the auth level used when issuing the eid", () => {
+    const globalState = appReducer(undefined, applicationChangeState("active"));
+
+    expect(
+      itwAuthLevelSelector(
+        _.set(globalState, "features.itWallet.preferences", {
+          authLevel: "L2"
+        })
+      )
+    ).toEqual("L2");
+    MockDate.reset();
```

Comment:
```suggestion
```
I'd also suggest to remove the `_.set`, which could lead to problems and unwanted behaviors within tests

Reply:
Thanks @mastro993, I improved the test using the spread operator and added another small test. Anyway IMO if we want to improve these test suites by replacing the usage of `lodash`, we should refactor every test (at least these) by removing it.